libs/ml/tests/unit/clustering/tsne_tests.cpp

```diff
@@ -104,48 +104,24 @@ TYPED_TEST(TsneTests, tsne_test_2d)
   // since tsne is a training procedure the number of operations is relatively large.
   // here we use 50 as proxy instead of calculating the number of operations
   // 50 is quite strict since there are 100 data points
-  EXPECT_NEAR(double(output_matrix.At(0, 0)), 0.25323880254290997982025146484375,
+  EXPECT_NEAR(double(output_matrix.At(0, 0)), 2.5455484559746151,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(1, 0)), -3.17587922653183341026306152343750,
+  EXPECT_NEAR(double(output_matrix.At(1, 0)), -1.7767733335494995,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(0, 25)), -1.75773554784245789051055908203125,
+  EXPECT_NEAR(double(output_matrix.At(0, 25)), 0.059521886824643898,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(1, 25)), 2.62657403736375272274017333984375,
+  EXPECT_NEAR(double(output_matrix.At(1, 25)), 2.8227819671468208,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(0, 50)), 0.23583724093623459339141845703125,
+  EXPECT_NEAR(double(output_matrix.At(0, 50)), -1.0112856793691054,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(1, 50)), 1.67974892887286841869354248046875,
+  EXPECT_NEAR(double(output_matrix.At(1, 50)), -0.057417890948507175,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(0, 99)), -0.87261968106031417846679687500000,
+  EXPECT_NEAR(double(output_matrix.At(0, 99)), 2.7302324351584537,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
-  EXPECT_NEAR(double(output_matrix.At(1, 99)), 3.04633120773360133171081542968750,
+  EXPECT_NEAR(double(output_matrix.At(1, 99)), 0.48101261687371411,
               50 * static_cast<double>(math::function_tolerance<DataType>()));
 }
 
-TYPED_TEST(TsneTests, tsne_test_2d_cross_type_consistency_test)
```

Comment:
Why is this test removed?

Reply:
As discussed, it's a duplicate of the test above!